### PR TITLE
Refactor 452 create pallet of color variable at globals.ts

### PR DESCRIPTION
**Pull request by 9Syncopation**
create color pallet variable names at _globals.ts

### CODE REVIEW

**Comment by 9Syncopation** [comment]
need to discuss white-gray pallet,
 it has some small differences that could be combined to few variables

File: src/components/atoms/AnywayLink.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
 import React, { FC } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import { Link, LinkProps } from 'react-router-dom';
-import { onLinkColor } from '../../style';
-import { onLinkColorHover } from '../../style';
+import { oceanBlue } from '../../style';
```
Comment (yuvalbl) [inline src/components/atoms/AnywayLink.tsx:4]: import in same line

File: src/components/atoms/SignInIcon.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { createStyles, makeStyles, Theme } from '@material-ui/core';
-import { signInIconColor } from '../../style';
+import { blackColor } from '../../style';
```
Comment (yuvalbl) [inline src/components/atoms/SignInIcon.tsx:3]: why "Color"?

File: src/components/atoms/TooltipMarker.tsx [hunk 3]
```diff
@@ -5,12 +5,7 @@ import { Marker } from 'react-leaflet';
 import { makeStyles } from '@material-ui/core';
 import { dateFormat } from '../../utils/time.utils';
 import { ClockPosition } from '../../utils/enum.utils';
-import {
-  tooltipMarkerBackgroundColorIcon,
-  tooltipMarkerColorContent,
-  tooltipMarkerBackgroundContent,
-  tooltipMarkerBorderColorArrow,
-} from '../../style';
+import { noColor, whiteColor, blackColor, tooltipMarkerBorderColorArrow } from '../../style';
```
Comment (yuvalbl) [inline src/components/atoms/TooltipMarker.tsx:8]: noColor => transparentColor

File: src/components/molecules/DialogWithHeader.tsx [hunk 4]
```diff
@@ -15,12 +15,12 @@ const useStyles = makeStyles({
   bar: {
     display: 'flex',
     width: 'inherit',
-    color: primaryColor,
+    color: primaryBlueViolet,
```
Comment (yuvalbl) [inline src/components/molecules/DialogWithHeader.tsx:18]: remove primary

File: src/style/_globals.ts [hunk 5]
```diff
@@ -1,17 +1,27 @@
...
-export const lighterWidgetText = '#464a47';
-export const onLinkColor = '#0088cc';
-export const onLinkColorHover = '#00aaff';
-export const borderColor = '#D3D3D3';
-export const selectedNewsFlash = '#CECECE';
-export const closeButtonColor = '4c4c4c';
-export const lightBackgroundColor = '#e2e2e2f0';
+export const blackColor = '#000';
+export const whiteColor = '#ffffff';
+export const noColor = 'transparent';
```
Comment (yuvalbl) [inline src/style/_globals.ts:3]: transparentColor

**Review by yuvalbl — APPROVED** [review]

**Pull request merged**